yt_dlp/extractor/tiktok.py

```diff
@@ -151,7 +152,8 @@ def _get_subtitles(self, aweme_detail, aweme_id):
         subtitles = {}
         # aweme/detail endpoint subs
         captions_info = traverse_obj(
-            aweme_detail, ('interaction_stickers', ..., 'auto_video_caption_info', 'auto_captions', ...), expected_type=dict)
+            aweme_detail, ('interaction_stickers', ..., 'auto_video_caption_info', 'auto_captions', ...),
+            expected_type=dict)
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
            aweme_detail, ('interaction_stickers', ..., 'auto_video_caption_info', 'auto_captions', ...), expected_type=dict)
```